subjects/tetrisoptimizer.audit.en.md

```diff
@@ -0,0 +1,62 @@
+`ERROR`
+###### Does the program prints the value above?
+##### Try bad example 01 (github.com/public/subject/badexample01.md).
+`ERROR`
+###### Does the program prints the value above?
+##### Try bad example 02 (github.com/public/subject/badexample02.md).
+`ERROR`
+###### Does the program prints the value above?
+##### Try bad example 03 (github.com/public/subject/badexample03.md).
+`ERROR`
+###### Does the program prints the value above?
+##### Try bad example 04 (github.com/public/subject/badexample04.md).
+`ERROR`
+###### Does the program prints the value above?
+##### Try bad format (github.com/public/subject/badformat.md).
+`ERROR`
+###### Does the program prints the value above?
+##### Try good example 00 (github.com/public/subject/goodexample00.md).
+###### Does the result contain 0 empty spaces (0 '.')?
+##### Try good example 01 (github.com/public/subject/goodexample01.md).
+###### Does the result contain 9 empty spaces (9 '.')?
+##### Try good example 02 (github.com/public/subject/goodexample02.md).
+###### Does the result contain 4 empty spaces (4 '.')?
+##### Try good example 03 (github.com/public/subject/goodexample03.md).
+###### Does the result contain 5 empty spaces (5 '.')?
+##### Try hard example (github.com/public/subject/hardexam.md).
+###### Does the result contain 1 empty spaces (1 '.')?
+###### Are all of the Tetrominos contained in the test file, present in the output?
+###### Different characters correspond to different Tetrominos?
+###### Does one Tetromino has only one character?
+
+#### Basic
+
+###### *Has the code passed the formatting? (gofmt, goimports)
+###### *Does the project runs quickly and effectively (Favoring of recursive, no unnecessary data requests, etc.)?
+
+###### *Does the program avoid the use of packages?
+###### *Does the code present commenting? (github.com/01-edu/public/blob/master/subjects/good-practices.en.md)
+###### *Does the code present documentation?
+###### *Does it present consistent indentation?
+###### *Does the code avoid obvious comments (no over information)?
+###### *Does the code have a consistent naming scheme (camelCase, under_scores)?
+###### *Does the code obey the principles "DRY" (Don't Repeat Yourself) or "DIE" (Duplication is Evil)?
+###### *Does the code obey the principal "KISS" (Keep It Simple, Stupid)?
+###### *Does the code obey the principle "YAGNI" (You Aren’t Gonna Need It)?
+###### *Does the code obey the principle "SOC" (Separation of Concerns)?
+###### *Does the code avoid deep nesting (if in ifs, forest of ifs)?
+###### *Does the code present a good file and folder organization?
+###### *Does it present a good separation of Code and Data?
+###### *Does the go code follow the go recommendation? (https://github.com/01-edu/public/blob/master/subjects/good-practices.en.md)
+
+###### *Is there a test file for this code?
+###### *Are the tests checking each possible case?
+###### *Is the error handled, is it error free?
```

Comment:
Same thing, please move this in https://github.com/01-edu/public/blob/master/subjects/good-practices.en.md to avoid duplication